docs/tactics.md

```diff
@@ -971,6 +971,34 @@ end
 ```
 and likewise for `to_rhs`.
 
+### apply_congr
```

Comment:
`tactics.md` is deprecated and all the content has been copied to tactic doc entries as of #2114. In particular, anything added here won't appear on [the mathlib doc page on tactics](https://leanprover-community.github.io/mathlib_docs/tactics.html), and in the near future we'll replace all the text here with a link to that page.

I've made a suggestion below that adds this text as a tactic doc entry per the instructions [here](https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/doc.md#tactic-doc-entries).

Reply:
Also, master should be merged, otherwise the "generate docs" step of the build will fail.

Reply:
Ah, of course. Are we going to delete `tactics.md`? Presumably we should do this now.

Reply:
I'll try to open a PR that cleans up the markdown files tonight.